index.js

```diff
@@ -41,6 +41,61 @@ function ELKReporter(runner) {
   });
 
   runner.on('end', function() {
+      /*
+      * When Cypress test passes, it provides a test instance to add context using mochawesome/addContext.
+      * But when Cypress test fails, it does not provide test instance to add context.
+      * However it provides test instance when that specific test finishes.
+      * So to add context on test failure, replace test objects in 'failures' array with its equivalent object from 'tests' array.
+      * 
+      * 1. Iterate over failures array. Push failed tests with no context in a map by mapping title to test's index in failures array.
+      * 2. Iterate over tests array, if map has that test title, replace the test object in failures with its equivalent in tests array.
+      *
+      * e.g. tests array - 
+      * [
+      *   Test { title: '[uuid:cy-template-00001], should do first thing', state: 'passed', context: {title: 'some string', value: {shopperId: '1111'}}, .....},
```

Comment:
This is very Godaddy specific detail which may not be relevant for the general audience who use this package.